test/kixi/stats/core_test.cljc

```diff
@@ -172,6 +172,29 @@
           (when-not (zero? d)
             (/ (reduce + (map * mxs mys)) d)))))))
 
+(defn cramers-v'
+  [fx fy coll]
+  (let [count-values (fn [m]
+                       (into {} (for [[k v] m] [k (count v)]) ))
+        x-counts (count-values (group-by fx coll))
```

Comment:
My sense is that `{x|y|xy}-counts` could be calculated in a single `reduce` which could replace the 6 iterations with 1